### PR TITLE
fix: modal footer action button display

### PR DESCRIPTION
**Pull request by mmataja**
References [link the ticket here]

## Motivation and context
Move action button to start position instead of end

## Before
image

## After
image

### CODE REVIEW

File: src/components/modal/index.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ const Modal: FC<PropsWithChildren<Props>> = ({
               justifyContent={
                 secondaryActionButton && primaryActionButton
                   ? 'space-between'
-                  : 'flex-end'
+                  : 'flex-start'
```
Comment (GutierrezGo) [inline src/components/modal/index.tsx:74]: did you check this change with all the UX-designers? this change will be in many places around the projects

Comment (GutierrezGo) [reply src/components/modal/index.tsx:74]: Do you know what is the reason behind this change?

Comment (mmataja) [reply src/components/modal/index.tsx]: Yea, Waiting for confirmation on that one

Comment (mmataja) [reply src/components/modal/index.tsx]: I have Green light!

Comment (mmataja) [reply src/components/modal/index.tsx]: > Do you know what is the reason behind this change?

Oh, the reason is the latest design of saved searches modal which uses modal footers for action buttons.
https://www.figma.com/file/L4sLqmHGj2b08MUbWDA07g/Search-Results-%26-Advanced-Search-(Explorations)?node-id=4898%3A120426&mode=dev

**Pull request merged**